roxie/udplib/udptrs.cpp

```diff
@@ -414,6 +438,10 @@ class CSendManager : implements ISendManager, public CInterface
         {
             if (udpTraceLevel > 0)
                 DBGLOG("UdpSender: send_resend_flow started");
+            // mck - should we match send_receive_flow thread (RR) priority ?
+#ifdef __linux__
+            setLinuxThreadPriority(2);
```

Comment:
Need to be very careful with thread priorities and unwanted effects - in particular priority inversion. Unless you have evidence that this made a positive difference, I'm not sure it's wise to change

Reply:
I agree.  I thought it should be RR policy but the value is hard to determine.